.eslintrc

```diff
@@ -11,7 +11,8 @@
   },
   // Additional global variables your script accesses during execution
   "globals": {
-    "Joomla": true
+    "Joomla": true,
+    "Cropper": true
```

Comment:
We should not allow the Cropper global in all scripts, because we are not loading it globally and we use it only in one plugin of the media manager. Instead we can use linline comments to allow the usage only where we know cropper is loaded. So we should add `/* global Cropper */` at the top of the crop.es6.js file.